index.js

```diff
@@ -13,22 +13,19 @@ let attempts = 0;
 const maxNumberOfAttempts = 5;
 
 // Returns a random number from min (inclusive) to max (exclusive)
-// Usage:
-// > getRandomNumber(1, 50)
-// <- 32
-// > getRandomNumber(1, 50)
-// <- 11
 function getRandomNumber(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
 function checkGuess() {
   // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  attempts++; // Increment attempts before checks
 
   hideAllMessages();
 
+//else block was missing and unsupported token '!=='
+
```

Comment:
Not sure I follow this comment. `!==` is a strict inequality operator in JavaScript.